src/main/scala/com/microsoft/hyperspace/index/plananalysis/PlanAnalyzer.scala

```diff
@@ -142,7 +142,7 @@ object PlanAnalyzer {
           _,
           _,
           _) =>
-        usedPaths += location.rootPaths.head.toString
+        usedPaths += location.rootPaths.head.getParent.toString
```

Comment:
can you explain why this change was needed?

Reply:
this function is used in `explain` api output generation, specifically the lines after 
```
Indexes Used:
<index name> : <index path>
```

Before this PR, `location.rootPaths.head` points to the directory where index files are present.
After this PR, `location.rootPaths` points to the list of all files which comprise the index.

For now, I have chosen to display the directory name by taking the parent of the rootPaths.head file. We can decide to plan the expected behavior or if explain needs to be redesigned. Please let me know your thoughts @imback82 .
Thanks

Reply:
https://github.com/microsoft/hyperspace/issues/141

Thanks @imback82 , done